utils/common/component.py

```diff
@@ -758,32 +758,29 @@ def max_plate_height(self):
 
 class Weld:
 
-    def __init__(self, material_g_o="",type="fillet", fabrication=KEY_DP_WELD_FAB_SHOP):
+    def __init__(self, material_g_o="",type="KEY_DP_WELD_TYPE_FILLET", fabrication=KEY_DP_WELD_FAB_SHOP):
```

Comment:
KEY_DP... is a variable. Please remove the quotes .

Reply:
Done.